### PR TITLE
Vk typescript example

### PR DESCRIPTION
**Pull request by valentyn88**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/cypress/cucumber.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+id: cyp-cucumber
+title: "Cucumber"
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/cypress/cucumber.md:3]:
```suggestion
title: Cucumber with Cypress
```

File: docs/testrunner-toolkit/configuration/espresso/cucumber.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-id: cucumber
+id: esp-cucumber
 title: "Cucumber"
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/espresso/cucumber.md:3]:
```suggestion
title: Cucumber with Espresso
```

File: docs/testrunner-toolkit/typescript.md [hunk 3]
```diff
@@ -4,7 +4,8 @@ title: Using TypeScript Tests
 sidebar_label: Using TypeScript
 ---
 
-If you wrote your automated tests using TypeScript in either Playwright, Puppeteer, or Cypress, you need to transpile your Typescript files to Javascript before running them with Testrunner Toolkit.
+If you wrote your automated tests using TypeScript in either Playwright ,or Puppeteer, you need to transpile your Typescript files to Javascript before running them with Testrunner Toolkit.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/typescript.md:7]:
```suggestion
If you wrote your automated tests using TypeScript in either Playwright or Puppeteer, you need to transpile your Typescript files to Javascript before running them with `saucectl`.
```

File: docs/testrunner-toolkit/typescript.md [hunk 4]
```diff
@@ -58,22 +57,28 @@ Except for the TestCafe image, these `.ts` files cannot run directly on any Test
...
-     - tests/
    suites:
      - name: "basic test"
-       match: ".*.(spec|test).js"
+       testMatch: 'tests/*.js'
    ```
    
    By default `saucectl` will pickup any `.js` files located in the designated directory, however with the `suites` field you can set more granular control with regular expressions.
 
 5. Finally, run `saucectl` to execute your TypeScript tests:
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/typescript.md:70]: This statement seems a bit confusing to me....aren't you technically running the JavaScript tests you just transpiled, NOT the Typescript tests?

File: sidebars.js [hunk 5]
```diff
@@ -545,12 +545,27 @@ module.exports = {
...
+                            'testrunner-toolkit/configuration/cypress/cucumber',
+                        ]
+                    },
                     'testrunner-toolkit/configuration/playwright',
                     'testrunner-toolkit/configuration/puppeteer',
                     'testrunner-toolkit/configuration/testcafe',
-                    'testrunner-toolkit/configuration/espresso',
-                    'testrunner-toolkit/configuration/cucumber',
+                    {
+                        type: 'catagory',
```
Comment (sweeneyskirt-sl) [inline sidebars.js:561]:
```suggestion
                        type: 'category',
```

File: sidebars.js [hunk 6]
```diff
@@ -545,12 +545,27 @@ module.exports = {
                 items: [
                     'testrunner-toolkit/configuration',
                     'testrunner-toolkit/configuration/common-syntax',
-                    'testrunner-toolkit/configuration/cypress',
+                    {
+                        type: 'catagory',
```
Comment (sweeneyskirt-sl) [inline sidebars.js:549]:
```suggestion
                        type: 'category',
```

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]

File: sidebars.js [hunk 7]
```diff
@@ -545,12 +545,27 @@ module.exports = {
...
                     'testrunner-toolkit/configuration',
                     'testrunner-toolkit/configuration/common-syntax',
-                    'testrunner-toolkit/configuration/cypress',
+                    {
+                        type: 'category',
+                        label: 'Cypress',
+                        collapsed: true,
+                        items: [
+                            'testrunner-toolkit/configuration/cypress',
+                            'testrunner-toolkit/configuration/cypress/cucumber',
```
Comment (sweeneyskirt-sl) [inline sidebars.js:554]:
```suggestion
                            'testrunner-toolkit/configuration/cypress/cyp-cucumber',
```

File: sidebars.js [hunk 8]
```diff
@@ -545,12 +545,27 @@ module.exports = {
...
                     'testrunner-toolkit/configuration/testcafe',
-                    'testrunner-toolkit/configuration/espresso',
-                    'testrunner-toolkit/configuration/cucumber',
+                    {
+                        type: 'category',
+                        label: 'Espresso',
+                        collapsed: true,
+                        items: [
+                            'testrunner-toolkit/configuration/espresso',
+                            'testrunner-toolkit/configuration/espresso/cucumber',
```
Comment (sweeneyskirt-sl) [inline sidebars.js:566]:
```suggestion
                            'testrunner-toolkit/configuration/espresso/esp-cucumber',
```

File: docs/testrunner-toolkit/typescript.md (hunk 4 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/typescript.md:70]: You are right, we are running JavaScript tests. I left this part as it was before. How do to suggest rewriting it?

Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/typescript.md:70]:
```suggestion
5. Finally, run `saucectl` to execute your transpiled tests:
```

Good?

**Comment by sweeneyskirt-sl** [comment]
Hey @valentyn88 -- I think you overwrote my direct commits from earlier with your force push? The build is failing again due to the sidebars file not being correct.

File: sidebars.js (hunk 6 above)
Comment (sweeneyskirt-sl) [inline sidebars.js:549]:
```suggestion
                        type: 'category',
```

File: sidebars.js [hunk 9]
```diff
@@ -545,12 +545,27 @@ module.exports = {
...
                     'testrunner-toolkit/configuration',
                     'testrunner-toolkit/configuration/common-syntax',
-                    'testrunner-toolkit/configuration/cypress',
+                    {
+                        type: 'catagory',
+                        label: 'Cypress',
+                        collapsed: true,
+                        items: [
+                            'testrunner-toolkit/configuration/cypress',
+                            'testrunner-toolkit/configuration/cypress/cucumber',
```
Comment (sweeneyskirt-sl) [inline sidebars.js:554]:
```suggestion
                            'testrunner-toolkit/configuration/cypress/cyp-cucumber',
```

File: sidebars.js [hunk 10]
```diff
@@ -545,12 +545,27 @@ module.exports = {
...
+                        ]
+                    },
                     'testrunner-toolkit/configuration/playwright',
                     'testrunner-toolkit/configuration/puppeteer',
                     'testrunner-toolkit/configuration/testcafe',
-                    'testrunner-toolkit/configuration/espresso',
-                    'testrunner-toolkit/configuration/cucumber',
+                    {
+                        type: 'catagory',
```
Comment (sweeneyskirt-sl) [inline sidebars.js:561]:
```suggestion
                        type: 'category',
```

File: sidebars.js [hunk 11]
```diff
@@ -545,12 +545,27 @@ module.exports = {
...
+                        label: 'Espresso',
+                        collapsed: true,
+                        items: [
+                            'testrunner-toolkit/configuration/espresso',
+                            'testrunner-toolkit/configuration/espresso/cucumber',
```
Comment (sweeneyskirt-sl) [inline sidebars.js:566]:
```suggestion
                            'testrunner-toolkit/configuration/espresso/espcucumber',
```

File: docs/testrunner-toolkit/configuration/cypress/cucumber.md (hunk 1 above)
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/cypress/cucumber.md:3]:
```suggestion
title: Cucumber with Cypress
```

File: docs/testrunner-toolkit/configuration/espresso/cucumber.md (hunk 2 above)
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/espresso/cucumber.md:3]:
```suggestion
title: Cucumber with Espresso
```

File: docs/testrunner-toolkit/typescript.md (hunk 3 above)
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/typescript.md:7]:
```suggestion
If you wrote your automated tests using TypeScript in either Playwright or Puppeteer, you need to transpile your Typescript files to Javascript before running them with Testrunner Toolkit.
```

File: sidebars.js [hunk 12]
```diff
@@ -545,12 +545,27 @@ module.exports = {
...
                     'testrunner-toolkit/configuration/testcafe',
-                    'testrunner-toolkit/configuration/espresso',
-                    'testrunner-toolkit/configuration/cucumber',
+                    {
+                        type: 'category',
+                        label: 'Espresso',
+                        collapsed: true,
+                        items: [
+                            'testrunner-toolkit/configuration/espresso',
+                            'testrunner-toolkit/configuration/espresso/espcucumber',
```
Comment (sweeneyskirt-sl) [inline sidebars.js:566]:
```suggestion
                            'testrunner-toolkit/configuration/espresso/esp-cucumber',
```

**Comment by sweeneyskirt-sl** [comment]
Okay. Tests are back to passing!

File: docs/testrunner-toolkit/typescript.md (hunk 4 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/typescript.md:70]: Yeap

Comment (valentyn88) [reply docs/testrunner-toolkit/typescript.md]: done

**Comment by valentyn88** [comment]
> Hey @valentyn88 -- I think you overwrote my direct commits from earlier with your force push? The build is failing again due to the sidebars file not being correct.

sorry for that

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**